.github/workflows/build.yml

```diff
@@ -129,13 +134,80 @@ jobs:
         echo "optflags=-O3 -fno-inline" >> $GITHUB_ENV
       if: matrix.name == 'debug'
 
+    - name: Install Clang 18 (debug asan)
```

Comment:
Are the `clang`(s) available on the runners not recent enough?

Reply:
Not in Ubuntu 22.04 and 20.04. 24.04's is.